strr-api/src/strr_api/services/registration_service.py

```diff
@@ -148,7 +148,7 @@ def create_registration(cls, user_id, sbc_account_id, registration_request: requ
         registration = Registration(
             user_id=user_id,
             sbc_account_id=sbc_account_id,
-            status=RegistrationStatus.PENDING,
+            status=RegistrationStatus.ACTIVE,
```

Comment:
With provisional application, registration will expire in a year from when it was created and not from when certificate is issued, is that correct?